infra/ballista/build/lambda-handler.py

```diff
@@ -0,0 +1,80 @@
+import os
+import subprocess
+import logging
+import sys
+import shutil
+import time
+
+
+logging.getLogger().setLevel(logging.DEBUG)
+__stdout__ = sys.stdout
+
+IS_COLD_START = True
+
+def handler(event, context):
+    """An AWS Lambda handler that runs the provided command with bash and returns the standard output"""
+    shutil.rmtree("/tmp", ignore_errors=True)
```

Comment:
did you try not wiping the state between runs (you can check out [databend](https://github.com/cloudfuse-io/lambdatization/blob/main/infra/databend/build/lambda-handler.py) to see how it is done)?

Reply:
I haven't. but is a good idea. I have implemented this now